packages/xdl/src/Analytics.ts

```diff
@@ -9,13 +9,19 @@ const PLATFORM_TO_ANALYTICS_PLATFORM: { [platform: string]: string } = {
   linux: 'Linux',
 };
 
-let _userId: string | undefined;
-let _userTraits: any;
-
 export class AnalyticsClient {
+  private _userId: string | undefined;
+  private _userTraits: any;
   private segmentNodeInstance: Segment | undefined;
   private version: string | undefined;
-  private userIdentifyCalled: boolean = false;
+
+  public get Verion() {
```

Comment:
I think you intended `Version` 😬  I think we also prefer lowercase, if that's conflicting with the private `version` class property, I think it's ok to use `_` prefix (although that's not the perfect solution)

Reply:
thanks for pointing that out!

I copied the styling from `.packages/expo-cli/build/credentials/context.js` and used `_property` for private and `property` for public getters.